action.yml

```diff
@@ -25,6 +25,10 @@ inputs:
     description: 'Tag of the built image'
     required: false
     default: ''
+  archs:
+    description: 'architecture to build on'
+    required: false
+    default: "amd64"
```

Comment:
I see that default value for arch in buildah is 'None'[1]. Would it be possible to use None/NULL/etc here too?


[1] https://github.com/redhat-actions/buildah-build#inputs-for-build-from-containerfile

Reply:
I have tested with amd64 which is the default value for x86 acrhitecture. i will retry with none operation

Reply:
After further private conversation I can see, that this option is OK, but will be glad if this is documented in readme too.